CMakeLists.txt

```diff
@@ -73,10 +73,10 @@ rapids_find_package(
 # add third party dependencies using CPM
 rapids_cpm_init()
 
-add_subdirectory(rapids_logger)
+CPMAddPackage(
+  NAME rapids_logger GITHUB_REPOSITORY rapidsai/rapids-logger GIT_SHALLOW FALSE GIT_TAG
+  1043e0f3989d75ad52f5212544b8154777e86fc9 VERSION 1043e0f3989d75ad52f5212544b8154777e86fc9)
```

Comment:
Are we going to have proper versions for this? How long will we need to use raw commit hashes?

Reply:
We will have proper versioning in place by 25.02. There are a couple of important things that I want to get done first like the changes for #1740 and setting up a rapids-cmake function for standardizing rapids-logger usage across all the repos. The goal is definitely to have everything done for this by 25.02.